app/src/main/java/com/teamagam/gimelgimel/app/view/BaseActivity.java

```diff
@@ -6,9 +6,19 @@
 import android.support.v7.app.AppCompatActivity;
 
 import com.teamagam.gimelgimel.R;
+import com.teamagam.gimelgimel.app.common.logging.LogWrapper;
+import com.teamagam.gimelgimel.app.common.logging.LogWrapperFactory;
 
 public abstract class BaseActivity<T extends Application> extends AppCompatActivity {
 
+    private static final LogWrapper sLogger = LogWrapperFactory.create(BaseActivity.class);
```

Comment:
would BaseActivity<T>.class work?
If not, we have to find a way to determine which activity is currently active. !resolve